app/models/search.rb

```diff
@@ -50,6 +52,10 @@ def geographical_area
     end
   end
 
+  def is_country_european?
```

Comment:
This method and its associate constant should be on `GeographicalArea` rather than the Search model

Reply:
100% agree with this one